### PR TITLE
Battle Royale buylist rework and tweaks

### PR DESCRIPTION
**Pull request by flappybatpal**
[BALANCE][QOL]

## About the PR 
1. Reworks the syndicate buylist to get include a br_allowed var.
2. Removes the following loot:
- **Wrestling Belt, Red Chainsaw, Cloaking Device**
- Nuclear operative uplink items
- Emagged hypospray
- Self charging powercell
- Banana grenade pouch
- Breaching Charge/Thermite Breaching Charge
- Duct tape
- Portapuke
- Cargo teleporter
- .38 AP Ammo
- Grenade starter kit
- Moustache grenade
- Syndicate hotdog cart
- Syndicate sauce
- Poison bottle/box
- Zappy microbomb
- Syringe gun
- Syndicate trash cart
- Cursed clown mask
- Boom boots
- Clown car
- Fake moustache
- Rev dagger (Syndicate dagger still in)
- Stealth storage
- Syndicate cigarette packet
- Radio jammer
- Sleepy pen
- Freedom implant
- Agent card
- Voice changer
3. Adds the following weapon loot:
- Nuclear Operative SMG
- Clock 188
- Nuclear Operative Grenade Launcher
- Gyrojet pistol
- Blaster pistol
- Green chainsaw
- Fragile sword
- Shuriken pouch
- Grenade pouches (Frag, High Explosive, Incendiary, Mixed Explosive, Banana)
- Beartrap pouch
4. Adds the following armor:
- Rocket boots
- Syndicate spacesuit
- Commissar greatcoat
- Knight armor
5. Removes hand teleporters, vehicles (except escape pod), secbots, portal generator.
6. Change loot chests to always hold a weapon with 50% chance of also holding armor.
7. Move shuttle call to 10 battlers remaining.
8. Try to fix shuttle timer.

## Why's this needed? 
1. Seems cleaner than a massive list. Avoids items defaulting to being allowed in BR, which caused issues when new stuff was added to the syndicate_buylist.
2. Wrestling Belt, Red Chainsaw, Cloaking Device are all game winning items that are just too safe to use and could combo together.
Nuclear operative uplink items were not meant to be in but slipped in after changes to uplinks.
Shotgun, .22 pistol and revolver have been moved to the locker/crate spawns. This was to prevent players getting a gun with multiple boxes of ammo.
Other items are removed due to not being combat focused, being too easy to remove a stunned player with (portapuke, trash cart) or being based around chemicals which are not fun for players in a combat mode.
This should result in more combat appropriate loot, 
3. More weapon variety.
4. More armor variety
5. Hand teleporters and vehicles could result in players leaving Z1 and being gibbed. Portal generator avoids someone accidentally killing themselves with it, Vehicles also promote hiding rather than fighting. Secbots didn't target due to captains ID but could be accidentally hit which was kinda annoying.
6. Should increase the amount of weapons around without losing out on armor. Makes the chests a bit more interesting.
7. Calling at 5 battlers left might be a bit late. It does take six minutes for the shuttle to arrive, several of them will die by the point it arrives.
8. Shuttle timer is meant to be 20 minutes rather than 2. Didn't work live, delayed a bit more just in case. Works locally...

## Changelog

```changelog
(u)Flappybat
(+)Many Battle Royale changes. Shuttle call moved to 10 players left. Cloaking device, Wrestling Belt, Red Saw removed. Whole load of mostly useless Syndicate items removed. More weapon and armor variety added. Loot chests now always hold a weapon and 50% chance of also some armor. Vehicles removed except escape pods. 
```

### CODE REVIEW

**Comment by Eagletanker** [comment]
I think the fun of BR is randomly getting full nukie kits and other guns. The portapuke and some of the intsakill stuff can go, but I dont think all the weapons and stuff need to go

**Comment by flappybatpal** [comment]
> I think the fun of BR is randomly getting full nukie kits and other guns. The portapuke and some of the intsakill stuff can go, but I dont think all the weapons and stuff need to go

Regarding the nukie stuff, I'm 99% sure it was just stuff from /datum/syndicate_buylist/commander/. So the uplink, bomb remote, backup requisition, laser designator etc. Maybe the empty class crate. Full kits wouldn't make it into the items list. Any nukie gear is from the other list of weapons/armor.

File: code/datums/gamemodes/battle_royale.dm [hunk 1]
```diff
@@ -172,11 +185,12 @@ var/global/list/datum/mind/battle_pass_holders = list()
 			someone_died++
 	if(someone_died && living_battlers.len <= 5)
 		command_alert("[living_battlers.len] battlers remain!","BATTLE STATUS ANNOUNCEMENT")
-		if (!emergency_shuttle.online)
-			emergency_shuttle.incall()
-			command_alert("The escape shuttle has been automatically called. Escape on the shuttle, kill everyone else or die!","Escape Shuttle")
 	else if(someone_died && living_battlers.len % 10 == 0)
 		command_alert("[living_battlers.len] battlers remain!","BATTLE STATUS ANNOUNCEMENT")
+	if(living_battlers.len <= 10)
```
Comment (ZeWaka) [inline code/datums/gamemodes/battle_royale.dm:190]: len bad!

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Comment by ZeWaka** [comment]
cat

**Review by ZeWaka — APPROVED** [review]
battle royale is an admin-only meme gimmick mode this is fine

**Pull request merged**